tools/biobb_pytorch/biobb_train_mdae.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="biobb_pytorch_train_mdae" name="TrainMdae" version="@TOOL_VERSION@" profile="22.05">
+    <description>Train a Molecular Dynamics AutoEncoder (MDAE) PyTorch model.</description>
+    <macros>
+      <token name="@TOOL_VERSION@">4.2.1</token>
+    </macros>
+
+    <requirements>
+      <requirement type="package" version="@TOOL_VERSION@">biobb_pytorch</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+
+      ln -s '$input_train_npy_path' ./input_train_npy_path.$input_train_npy_path.ext &&
+
+      #if $input_model_pth_path:
+        ln -s '$input_model_pth_path' ./input_model_pth_path.$input_model_pth_path.ext &&
+      #end if
+      #if $config_json:
+        ln -s '$config_json' ./config_json.$config_json.ext &&
+      #end if
+
+      train_mdae
+
+      #if $config_json:
+        --config ./config_json.$config_json.ext
+      #end if
+
+      --input_train_npy_path ./input_train_npy_path.$input_train_npy_path.ext
+      #if $input_model_pth_path:
+        --input_model_pth_path ./input_model_pth_path.$input_model_pth_path.ext
+      #end if
+      --output_model_pth_path ./output_model_pth_path.pth
+      #if $output_train_data_npz_path:
+        --output_train_data_npz_path ./output_train_data_npz_path.npz
+      #end if
+      #if $output_performance_npz_path:
+        --output_performance_npz_path ./output_performance_npz_path.npz
+      #end if
+      ;
+      ]]>
+    </command>
+
```

Comment:
```suggestion
    <configfiles>
        <configfile name="train_config">
{
  "properties": {
    "num_epochs": $num_epoch,
    "seed": $seed
  }
}
        </configfile>
    </configfiles>
```

This way you can create those configfiles on the fly and ask your users for the inputs